frontend/src/containers/RecipePage/Recipe.js

```diff
@@ -41,12 +43,17 @@ class Recipe extends Component {
         userId: response.data.user_id,
       });
     });
+    getRecipeTags(id).then((data) => {
```

Comment:
I think we should be able to modify the recipe `show` method in Rails to include the tags, so we don't need to make this second call at all.